homeassistant/components/redgtech/__init__.py

```diff
@@ -0,0 +1,35 @@
+import logging
+from homeassistant.core import HomeAssistant
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import Platform, CONF_ACCESS_TOKEN
+from .const import DOMAIN
+from .coordinator import RedgtechDataUpdateCoordinator
+from dataclasses import dataclass
+
+_LOGGER = logging.getLogger(__name__)
+
+@dataclass
+class RedgtechEntryData:
+    coordinator: RedgtechDataUpdateCoordinator
+
+PLATFORMS: list[Platform] = [Platform.SWITCH]
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Set up Redgtech from a config entry."""
+    _LOGGER.debug("Setting up Redgtech entry: %s", entry.entry_id)
+    coordinator = RedgtechDataUpdateCoordinator(hass, entry)
+
+    await coordinator.async_config_entry_first_refresh()
+
+    hass.data.setdefault(DOMAIN, {})[entry.entry_id] = RedgtechEntryData(
+        coordinator=coordinator
+    )
```

Comment:
I already told you to to remove that
```suggestion
    entry.runtime_data = RedgtechEntryData(
        coordinator=coordinator
    )
```